[BUG] - openssl/evp.h' file not found when compiling on MacBook Pro ARM

**Describe the bug**
I've followed the procedure described here:
- https://github.com/Haivision/srt/blob/master/docs/build/build-macOS.md

But got this error:
```
In file included from /Users/jmgonet/Documents/Dev/srt/haicrypt/cryspr.c:20:
In file included from /Users/jmgonet/Documents/Dev/srt/haicrypt/hcrypt.h:48:
In file included from /Users/jmgonet/Documents/Dev/srt/haicrypt/cryspr.h:39:
In file included from /Users/jmgonet/Documents/Dev/srt/haicrypt/cryspr-config.h:13:
/Users/jmgonet/Documents/Dev/srt/haicrypt/cryspr-openssl-evp.h:22:10: fatal error: 'openssl/evp.h' file not found
#include <openssl/evp.h> /* PKCS5_xxx() */
```

**To Reproduce**
Steps to reproduce the behavior:

1. On a MacBook Pro, with ARM architecture (see description below)
2. Using this installation procesure:
```bash
brew update
brew upgrade
brew install srt
brew install openssl
brew install cmake
export OPENSSL_ROOT_DIR=$(brew --prefix openssl)
export OPENSSL_LIB_DIR=$(brew --prefix openssl)"/lib"
export OPENSSL_INCLUDE_DIR=$(brew --prefix openssl)"/include"
git clone https://github.com/Haivision/srt.git
cd srt
./configure
make
```
3. Compilation works until 51% is reached, and then the compilation error described above shows up.

**Expected behavior**
When following the procedure it should compile.

**Desktop (please provide the following information):**
 - Computer: MacBook Pro
 - Chip: Apple M3 Pro
 - OS: Sonoma 14.6.1
 - SRT Version / commit ID: 
   - master / e2f133b95c06dd3c157b1669de34b2432e640029
   - Also tried with v1.5.3, with same result.

--- a/docs/build/build-macOS.md
+++ b/docs/build/build-macOS.md
@@ -19,6 +19,7 @@ Install [CMake](https://cmake.org/) and OpenSSL with development files from `bre
 ```shell
 brew install cmake
 brew install openssl
+brew install pkgconfig
 ```
 
 SRT can be now built with `cmake` or `make` on Mac.

